caf-audit-service-client/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.github.cafaudit</groupId>
+    <artifactId>caf-audit-service-client</artifactId>
+    <version>2.1.0-SNAPSHOT</version>
```

Comment:
audit-service is an aggregator project so this child POM must match the parent which is currently sitting at 2.1.0-SNAPSHOT.